doc/getting_started/toolchain_3rd_party_x_compilers.rst

```diff
@@ -11,13 +11,16 @@ external organization. Several of these are available.
 GNU ARM Embedded
 ****************
 
+   .. warning::
+
+      Do not install the toolchain into a path with spaces.
+
 #. Download and install a `GNU ARM Embedded`_ build for your operating system
    and extract it on your file system.
 
    .. warning::
 
-      Do not install the toolchain into a path with spaces. On
-      Windows, we'll assume you install into the directory
+      On Windows, we'll assume you install into the directory
```

Comment:
Should this still be a `warning` ?
I would say that with the space info moved to the top, then this could be:
```
.. note::

      On Windows, we'll assume you install into the directory
```